sdk/src/program_error.rs

```diff
@@ -5,7 +5,7 @@ use thiserror::Error;
 
 #[cfg(feature = "program")]
 use crate::info;
-#[cfg(not(feature = "program"))]
+#[cfg(all(feature = "everything", not(feature = "program")))]
```

Comment:
Was this the underlying motivation for this change?

Reply:
Not directly.  When trying to test stuff over in SPL land, the semantics of enabling "program" causing code to be removed was generating confusing errors.